CMakeLists.txt

```
@@ -53,7 +53,7 @@ set(bifcl_SRCS
 add_executable(bifcl ${bifcl_SRCS})
 
 if (MSVC)
-    target_link_libraries(bifcl PRIVATE zeek_windows)
+    target_link_libraries(bifcl PRIVATE libunistd)
```

```suggestion
    # If building separately from zeek, we need to add the libunistd subdirectory so
    # that linking doesn't fail.
    if ("${CMAKE_PROJECT_NAME}" STREQUAL "BifCl")
        add_subdirectory(auxil/libunistd EXCLUDE_FROM_ALL)
    endif()

    target_link_libraries(bifcl PRIVATE libunistd)
```

This is needed so that bifcl will continue building separately from Zeek, but won't cause Zeek's build to have duplicate library issues.

---

done